src/components/MapPage/index.js

```diff
@@ -25,14 +26,16 @@ const MapPage = () => {
     fetchLocations();
   };
 
-  const getUserLocation = () => {
-    window.navigator.geolocation.getCurrentPosition((position) => 
-      setUserLocation([position.coords.latitude, position.coords.longitude])
+  const getUserLocation = async () => {
+    await window.navigator.geolocation.getCurrentPosition(async (position) => 
+      await setUserLocation([position.coords.latitude, position.coords.longitude])
     );
+    fetchLocations(userLocation)
   };
 
   const fetchLocations = () => {
-    console.log('Fetchy fetch fetch')
+    const url = 'https://cohelp-backend.herokuapp.com/api/v1/locations/sort?lat=39.7504&lng=-104.9963'
+    fetchLocationsCall(url)
   };
```

Comment:
We just need to remember to interpolate the `lat`, `lng` figures later. This worked for stand in testing of our connection to the backend.